graph_def_editor/node.py

```diff
@@ -120,19 +132,31 @@ def id_in_graph(self) -> int:
     """
     return self._id
 
+  def remove_from_graph(self):
```

Comment:
This method was only public because I hadn't figured out how to suppress warnings in IntelliJ. Did some poking around and discovered the magic incantation, `# noinspection PyProtectedMember`.